msteams-platform/graph-api/meeting-transcripts/get-transcripts-api.md

```diff
@@ -0,0 +1,891 @@
+| As an HR initiative, you're holding a brainstorming session to understand and improve employee health and productivity. Having to continually take notes to provide post-meeting summary can impede the flow of thoughts. You stand to lose useful valuable or suggestions. After the session, you'd need to analyze the discussion to gather data points for planning improvements. | Using Graph APIs to fetch transcripts post-meeting frees you and the participants to fully focus on the discussion. The content of the meeting transcript is available for: <br> • Engagement and sentiment analysis <br> • Listing tasks or issues <br> • Follow-up meetings and notifications |
+
+To fetch the transcript for a particular meeting:
+
+- [Configure required permissions for your app on Azure AD portal](#configure-permissions-to-access-transcript)
+- [Obtain meeting ID and the user ID of the meeting organizer](#obtain-meeting-id-and-organizer-id)
+- [Use Graph APIs to fetch transcript](#use-graph-apis-to-fetch-transcript)
+
+## Configure permissions to access transcript
+
+Your app must have the required permissions for fetching transcripts. Your app can access and fetch transcripts for a Teams meeting using classic permissions or RSC permissions.
+
+### Use classic permissions
+
+You can configure your app to access meeting transcripts across the tenant where it's installed. In this case, the meeting organizer doesn't need to install your app in the Teams meeting chat. Because of tenant-wide classic permissions authorized by tenant administrator, your app can read and access transcripts for all meetings in the tenant.
+
+The following permissions can be granted to your app:
+
+| Permission | Display name | Description |
+| --- | --- | --- |
+| OnlineMeetingTranscript.Read.All in Application context | Read the transcript of the meeting. | It allows the app to read meeting transcripts in your organization. It needs the administrator to consent, and doesn't require an app user to be signed in. |
+| OnlineMeetingTranscript.Read.All in Delegated (work or school account) context | Read the transcript of the meeting. | It allows your app to read meeting transcripts in your organization for a signed-in app user. It needs the administrator to consent. |
+
+/ reference to article for configuring classic permissions. /
+/ reference to article for admin page. /
+
+### Use RSC permissions
+
+If you want your app to fetch transcripts from a specific Team meeting, configure RSC permission for your app. The meeting organizer can install your app in the Teams meetings chat. After the meeting is closed, your app can make the API call to obtain the transcript for that meeting.
```

Comment:
`If you want your app to fetch transcripts from a specific Team meeting, configure RSC permission for your app.`

Probably mention like: 'If you want your app to access only the Team meetings that the app is installed in, configure RSC permission for your app.'